semgrep-core/tests/scala/import_metavariable.scala

```diff
@@ -0,0 +1 @@
+import A.B, C.D
```

Comment:
Related to my comment above, I think this test would fail if Semgrep actually matched on this line. If you expect a match you need to precede it with `// MATCH:` or similar.